playbooks/kind-integration-test-arm64/run.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - install-docker-ce
```

Comment:
`install-docker-ce` should be updated to support ARM arch, like I say in previous comments.
1. support ARM arch in https://github.com/theopenlab/openlab-zuul-jobs/blob/master/roles/install-docker-ce/tasks/main.yml#L24
2. cgroupdriver is not supported in ARM arch, should be updated in https://github.com/theopenlab/openlab-zuul-jobs/blob/master/roles/install-docker-ce/tasks/main.yml#L40
So I believe docker can not be launched in current job define.